src/main/java/com/hedera/hashgraph/sdk/account/AccountAddClaimTransaction.java

```diff
@@ -19,7 +19,7 @@ public AccountAddClaimTransaction(Client client) {
         super(client);
     }
 
-    AccountAddClaimTransaction() {
+    public AccountAddClaimTransaction() {
```

Comment:
Why are all these constructors set to `public` here? These are for testing only.

Reply:
Making it possible to create an unsigned transaction without a client